docs/resources/artifactory_keypair.md

```diff
@@ -35,13 +35,21 @@ resource "artifactory_keypair" "some-keypair6543461672124900137" {
 
 The following arguments are supported:
 
-* `pair_name` - (Required) name of the key pair and the identity of the resource.
-* `pair_type` - (Required) RT requires this - presumably for verification purposes.
-* `alias` - (Required) Required but for unknown reasons
-* `private_key` - (Required)  - duh! This will have it's pem format validated
-* `passphrase` - (Optional/Questionable)  - This will be used to decrypt the private key. Validated server side.
-* `public_key` - (Required)  - duh! This will have it's pem format validated
-* `unavailable` - (Computed) - it's unknown what this does, but, it's returned in the payload and there is no known place to set it in the UI
+* `pair_name` - (Required) A unique identifier for the Key Pair record.
+* `pair_type` - (Required) Artifactory requires this - presumably for verification purposes. ?????
```

Comment:
This is not good description. The description in the schema is better but also can be improved. For example, we now support both RSA and GPG types.

Reply:
Agree